internal/template/utilization_functions.go

```diff
@@ -68,10 +63,21 @@ scalar(
     )
   * 100, 0.0001)
 )`
+)
 
+func cpuUtilization(data MetricData, labelArgs ...string) (string, error) {
+	return renderUtilization(cpuUtilizationQueryTemplate, data, labelArgs...)
+}
+
+func memoryUtilization(data MetricData, labelArgs ...string) (string, error) {
 	return renderUtilization(memoryUtilizationQueryTemplate, data, labelArgs...)
 }
 
+func avgUtilization(data MetricData, labelArgs ...string) (string, error) {
+	avgQuery := fmt.Sprintf("\n(%s\n+%s\n)\n/2", cpuUtilizationQueryTemplate, memoryUtilizationQueryTemplate)
```

Comment:
Just to be clear, without this it's just `query: '({{cpuUtilization . foo=bar}}+{{memoryUtilization . foo=bar}})/2'` right?

Reply:
I haven't tested doing both independently like that, but yeah that's what it boils down to.